src/data/resources.js

```diff
@@ -1,172 +1,581 @@
 const resources = [
     {
-        category: "category",
+        category: "HTML/CSS",
         links: [
             {
-                name: "Resource 1",
-                url: "https://github.com/uclaacm/dev-pathways",
+                name: "Hack 1",
```

Comment:
These names should be more descriptive of what the resource is. For example, for this resource a good name would be "Hack School: Intro to HTML/CSS."